kafka-connect-upload-source/src/main/java/org/radarbase/connect/upload/converter/altoida/summary/AltoidaTestMetricsProcessor.kt

```diff
@@ -18,201 +18,201 @@ class AltoidaTestMetricsProcessor(private val type: AltoidaTestCategory, val top
-                line.getTappingTestAggregate(prefix, RandomTapping),
-                line.getTappingTestAggregate(prefix, Tapping)
+                line.getValue("${prefixAR}HIGHTONEREACTIONTIMES").toFloat(),
+                line.getValue("${prefixAR}HIGHTONETOUCHACCURACY").toFloat(),
+                line.getValue("${prefixAR}NMRREACTIONSTOLOWTONE").toInt(),
+                line.getValue("${prefixAR}IGNOREDHIGHTONEPERCENTAGE").toFloat(),
+                line.getValue("${prefixAR}PREMATURETONEBUTTONPRESSES").toInt(),
+                line.getValue("${prefixAR}RANDOMSCREENPRESSESDURINGPLACEMENT").toInt(),
+                line.getValue("${prefixAR}RANDOMSCREENPRESSESDURINGSEARCH").toInt(),
+                line.getValue("${prefixAR}TOOMUCHMOVEMENTCOUNT").toInt(),
+                line.getValue("${prefixAR}FINDBETTERPLACECOUNT").toFloat(),
+                line.getValue("${prefixAR}INTROREADTIMES").toFloat(),
+                line.getValue("${prefixAR}INTROREADTIME1").toFloat(),
+                line.getValue("${prefixAR}INTROREADTIME2").toFloat(),
+                line.getValue("${prefixAR}PLACEDELAYSAVERAGE").toFloat(),
+                line.getValue("${prefixAR}SPOTALREADYTAKENCOUNT").toInt(),
+                line.getTrails(prefixAR),
+                line.getTrailMeans(prefixAR),
+                line.getValue("${prefixAR}FINDFAILCOUNT").toFloat(),
+                line.getValue("${prefixAR}FINDSKIPDURATIONS").toFloat(),
+                line.getValue("${prefixAR}SKIPBUTTONCOUNT").toFloat(),
+                line.getValue("${prefixAR}COUNTDOWNFAIL").toFloat(),
+                line.getValue("${prefixAR}STEPCOUNTFINDPLACERATIO").toFloat(),
+                line.getValue("${prefixAR}PLACINGMEANSTEPDELAY").toFloat(),
+                line.getValue("${prefixAR}FINDINGMEANSTEPDELAY").toFloat(),
+                line.getValue("${prefixAR}PLACINGSTEPVARIANCE").toFloat(),
+                line.getValue("${prefixAR}FINDINGSTEPVARIANCE").toFloat(),
+                line.getValue("${prefixAR}PLACINGNOTWALKINGTIME").toFloat(),
+                line.getValue("${prefixAR}FINDINGNOTWALKINGTIME").toFloat(),
+                line.getValue("${prefixAR}SHOCKCOUNT").toFloat(),
+                line.getValue("${prefixAR}ACCVARIANCEX").toFloat(),
+                line.getValue("${prefixAR}ACCVARIANCEY").toFloat(),
+                line.getValue("${prefixAR}ACCVARIANCEZ").toFloat(),
+                line.getValue("DOMINANTHAND").toFloat(),
+                line.getWalkingTestAggregate(prefixMotor, Circle),
+                line.getWalkingTestAggregate(prefixMotor, Square),
+                line.getWalkingTestAggregate(prefixMotor, Serpentine),
+                line.getWalkingTestAggregate(prefixMotor, SpeedCircle),
+                line.getTappingTestAggregate(prefixMotor, RandomTapping),
+                line.getTappingTestAggregate(prefixMotor, Tapping)
         )
     }
 
     private fun Map<String, String>.getTrails(prefix: String): List<AltoidaTrial> {
```

Comment:
By the way, I'm not sure, but should this be `getTrials` instead? (Also the means method?)

Reply:
not sure if I follow

Reply:
I'm just wondering if it's a typo or not?

Reply:
Ah. I see. Yes. It is indeed a typo (which obviously I can't see even when pointed out)